Source/Models/APIClient.swift

```diff
@@ -1,44 +0,0 @@
-import Foundation
-
-public struct APIClient {
-  fileprivate let requestPerformer: RequestPerformer
-  fileprivate let scheduler: Scheduler
-
-  public init(requestPerformer: RequestPerformer = NetworkRequestPerformer(), scheduler: @escaping Scheduler = mainQueueScheduler) {
```

Comment:
Oh hey I think these files are showing up as deleted + added because they've also been reindented from 2 to 4 spaces. What do you think about going back to 2 space indent?

This .swiftformat file seems to have reasonable results:

```
--exclude Carthage
--indent 2
--ifdef outdent
--self init-only
```